src/interfaceImplementsClass.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Vendor\Hw3;
+
+class interfaceImplementsClass implements iInterface
+{
+    public function setVariable($name, $var)
+    {
+        echo $name.'='.$var;
```

Comment:
the method is called "setVariable" but it does not sets anything, just prints the assignment expression.
I didn't understand what this class is doing. Why its name begins with  interface, but it is not interface. If your class prints the expressions, it will be better to return them as strings and print in your index.php